engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -356,6 +356,15 @@ paths:
             returns workflows with all of the specified label keys. Specify the label key
             and label value pair as separated with
             "label-key:label-value"
+        - name: additionalKeys
```

Comment:
Not 100% sure about the name. It isn't clear this is related to the query endpoint, and if it is understood to relate to the query endpoint it sounds like it allows for additional keys (parameters) to the query rather than additional key/value pairs in the results.

Reply:
Maybe `additionalQueryResultFields`? 🙂